src/js/components/Search.js

```diff
@@ -165,14 +165,16 @@ export default class Search extends Component {
       'bubbles': true,
       'cancelable': true
     });
-    this.refs.input.dispatchEvent(event);
-    this.props.onDOMChange(event.target.value, event);
+    var controlInput = document.getElementById('search-drop-input');
```

Comment:
This won't work if there are multiple Search components on a page. I think we need to still use the this.refs approach.